.github/workflows/ci-cd-trigger.yml

```diff
@@ -136,19 +137,28 @@ jobs:
             preview_explorer=$(printf "https://%s.%s.vega.rocks" "explorer" "$branch_slug")
           fi
           projects="$(echo $projects_e2e | sed 's|-e2e||g')"
-          if echo "$affected" | grep -q multisig-signer; then
-            echo "Tools are affected"
-            # tools are only applicable to check previews or deploy from develop to mainnet
-            if [[ "${{ github.event_name }}" = "pull_request" ]]; then
+          if [[ "${{ github.event_name }}" = "pull_request" ]]; then
```

Comment:
**thought (non-blocking)**:  Mirroring my thought on the other related PR: This logic is getting too big for a bash-embedded-in-yaml-script, but there is a separate ticket to address that, so I am reviewing assuming we'll move this out eventually